app/controllers/organizations/default_pet_tasks_controller.rb

```diff
@@ -1,7 +1,54 @@
 class Organizations::DefaultPetTasksController < Organizations::BaseController
   layout "dashboard"
 
+  before_action :verify_organization_for_current_user, only: [:new, :create, :edit, :update, :destroy]
```

Comment:
Any reason why you skipped the index? 🤔 I think we should also check it, we don't want to let users peek into tasks that were added in another org 😄 
```suggestion
  before_action :verify_organization_for_current_user
```